packages/dhis-api/src/__tests__/responseUtils.test.js

```diff
@@ -59,21 +57,17 @@ const createConflictsAndMessages = (count = 1) => {
 };
 
 describe('responseUtils', () => {
-  before(() => {
+  beforeAll(() => {
     // Suppress logging while running the tests
-    sinon.stub(winston, 'warn');
-  });
-
-  after(() => {
-    winston.warn.restore();
+    jest.spyOn(winston, 'warn').mockImplementation(() => {});
```

Comment:
Probably we don't need this block because we do this in `jestScaffolding.js`

Reply:
Good spot! I did miss that.